src/docs/30_mui_components/buttons.mdx

```diff
@@ -124,6 +124,13 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 		<LensToggle color="info" />
 		<LensToggle color="success" />
 		<LensToggle color="warning" />
-		<LensToggle color="error" />
+	</Box>
+	<Box sx={{ display: 'flex', gap: 2, mt: 2, flexWrap: 'wrap' }}>
+		<LensToggle color="standard">Toggle text</LensToggle>
```

Comment:
The hover color for the unselected one should be `bg.strong` instead of `bg.accent`